src/run_prettify.js

```diff
@@ -1662,7 +1665,7 @@ var IN_GLOBAL_SCOPE = false;
               'keywords': JSCRIPT_KEYWORDS,
               'cStyleComments': true,
               'regexLiterals': true
-            }), ['javascript', 'js']);
+            }), ['javascript', 'js', 'ts', 'typescript']);
```

Comment:
same thing, this is coming from an already merged PR #423, which modified the generated file `src/prettify.js` and not the actual source in `js-modules/prettify.js`.